README.md

```diff
@@ -1,32 +1,33 @@
-# Linuxbrew-fonts
+# Fonts for Homebrew on Linux
 
-Linuxbrew-fonts is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-fonts/).
-
-Linuxbrew-fonts is a Homebrew Tap which allows you to use the same friendly Homebrew-style CLI workflow for the administration of binary font files on your Linux.
+homebrew-linux-fonts is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-cask-fonts). 
+homebrew-linux-fonts is a Homebrew Tap which allows you to use the same friendly Homebrew-style CLI workflow for the administration of binary font files on your Linux.
+It is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-cask-fonts), since Homebrew Casks are only for macOS.
 
 ## Prerequisites
 
-Fonts are available through homebrew and linuxbrew. So, first you must install [Homebrew](http://brew.sh) or [Linuxbrew](http://linuxbrew.sh). However this repository is **NOT** an "external command” of Homebrew, it is a different from the upstream repository, homebrew/homebrew-cask-fonts. Please caught of that.
-
-**Since Homebrew 1.9 or later supports the both platform, you can use this repository with homebrew in Linux too**
+These fonts are installable with [Homebrew](http://brew.sh).
+This repository is **NOT** an "external command" of Homebrew.
+It is different from the upstream repository, `homebrew/homebrew-cask-fonts`.
+Please be aware of that.
```

Comment:
```suggestion
These fonts are installable with [Homebrew](http://brew.sh).
Once installed, you can configure this repo as a source for fonts.

Note that this repo differs from the upstream repository, `homebrew/homebrew-cask-fonts`.
The set of fonts available may not match exactly.
Moreover, it's best to use only one of these repos on macOS;
technically this repo _and_ the upstream are usable on macOS,
but you should not use _this repo_ on macOS because upstream will have the majority of what's needed.
Changes to upstream eventually propagate to this repo.
```

Thoughts? Or too long?

Reply:
I think lines 14–16 are not necessary, since this project already has "linux" in the name, and there's `homebrew/homebrew-cask-fonts` for macOS users anyway. Those two facts seem to me to be sufficient to signal that this is not meant for use in macOS.

Reply:
```suggestion
These fonts are installable with [Homebrew](http://brew.sh).
Once installed, you can configure this repo as a source for fonts.

Note that this repo differs from the upstream repository, `homebrew/homebrew-cask-fonts`.
The set of fonts available may not match exactly.
Changes to upstream eventually propagate to this repo.
```

Thoughts? Or too long?